bundles/AdminBundle/RateLimiter/AdminRequestRateLimiter.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\AdminBundle\RateLimiter;
+
+use Pimcore\Bundle\AdminBundle\Security\Authenticator\AdminSessionAuthenticator;
+use Symfony\Component\HttpFoundation\RateLimiter\RequestRateLimiterInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\RateLimiter\Policy\NoLimiter;
+use Symfony\Component\RateLimiter\RateLimit;
+
+/**
+ * Disables rate limit on the \Pimcore\Bundle\AdminBundle\Security\Authenticator\AdminSessionAuthenticator
```

Comment:
```suggestion
 * @internal
 * Disables rate limit on the \Pimcore\Bundle\AdminBundle\Security\Authenticator\AdminSessionAuthenticator
```
so that we can easily kick it in later version.